src/components/controlButtons/controlButtons.js

```diff
@@ -0,0 +1,52 @@
+import { createElement } from "../../utils/elements";
+
+export const ControlButtons = () => {
+  const OuterDIV = createElement("div", {
+    className: "nokiaControlButtons",
+  });
+
+  const numbersFirstRow = [{ name: "1" }, { name: "2" }, { name: "3" }];
+  const numbersSecondRow = [{ name: "4" }, { name: "5" }, { name: "6" }];
+  const numbersThirdRow = [{ name: "7" }, { name: "8" }, { name: "9" }];
+  const numbersFourthRow = [{ name: "*" }, { name: "0" }, { name: "#" }];
+
+  const firstRow = createElement("div");
+  const secondRow = createElement("div");
+  const thirdRow = createElement("div");
+  const fourthRow = createElement("div");
+
+  numbersFirstRow.forEach((buttons) => {
+    const buttonsbtn = createElement("button", {
+      innerText: buttons.name,
+    });
+    firstRow.append(buttonsbtn);
```

Comment:
So many buttons... Kinda confusing for me at the moment, but maybe it's too late in the afternoon :)